core/serialisation/data_serialiser.py

```diff
@@ -74,6 +74,30 @@ def _serialize(self, value, attr, obj, **kwargs):
         return ", ".join(value)
 
 
+class JSONBFloatField(fields.Float):
```

Comment:
could you also update the module docstring to include the JSONB stuff, including these new custom classes?